plugins/chat/assets/javascripts/discourse/services/chat-channel-thread-indicator-subscriptions-manager.js

```diff
@@ -0,0 +1,56 @@
+import Service, { inject as service } from "@ember/service";
```

Comment:
Im not totally sold on needing this. You are doing this because this logic is not present on the thread right? I think it doesn’t matter because it shouldn't receive this event right?

Even if it was I would probably prefer options when you subscribe. We already have a lot of services, they are handy but they also add indirection and complexity, if we can avoid it here that would be nice IMO.

Reply:
I'm adding this because there will be many thread indicators for the channel updating live, for example the counts, maybe excerpt, user avatars. However, maybe you are right, another service is not needed, and it could just live in the `ChatChannelPaneSubscriptionsManager` 👍